### PR TITLE
Update to 2.3

### PR DESCRIPTION
**Pull request by kaechele**
- changed upstream URL to use a sensible filename
- add lua_content directory
- use %%autosetup macro
- refresh ciphers patch
- add replxx compile fix patch

### CODE REVIEW

**Comment by LorbusChris** [comment]
Thank you!

One Question:
Is there an upstream reference for the new patch? I.e. why is it needed, and will it be fixed upstream? If there is no upstream issue, yet, I'd ask you to file one and reference it in the spec so we can track it.

**Comment by kaechele** [comment]
https://github.com/AmokHuginnsson/replxx/pull/65

Since it is bundled directly into rspamd we'd also need to inform them as soon as above PR is merged.

File: rspamd.spec [hunk 1]
```diff
@@ -104,8 +105,7 @@ with big amount of mail and can be easily extended with own filters written in
 lua.
 
 %prep
-%setup -q
-%patch0 -p1
+%autosetup -p1
```
Comment (LorbusChris) [inline rspamd.spec:108]: if you could add a link to the upstream replxx PR here, I'll merge :)

**Review by LorbusChris — CHANGES_REQUESTED** [review]

**Comment by kaechele** [comment]
2.4 is out. Will resubmit.

**Comment by LorbusChris** [comment]
thanks!

**Pull request closed without merging**